README.md

```diff
@@ -1,3 +1,164 @@
-# Model Zoo
+# Autoware Model Zoo
 
-The Autoware Model Zoo, containing machine-learned models for use in autonomous driving applications.
\ No newline at end of file
+A collection of machine-learned models for use in autonomous driving
+applications.
+
+## License
+
+All source code published in this repository by default are licensed under
+Apache 2.0. If contributions need to be made under a difference license, the
+LICENSE body need to be included in the sub-folder of the model. License for the
+model files can be specified using the license field in
+[definition.yaml](#filling-out-the-definition-file).
```

Comment:
It's very important that the license for the model be specified separately from the license for the training data. This is a major issue for commercial use of trained models.

Reply:
do you think we should have separate fields for model license and data license?

Reply:
I really do. It would help immensly for companies looking to use the models commercially.